ecs-init/docker/dependencies.go

```diff
@@ -96,3 +129,17 @@ var standardFS = &_standardFS{}
 func (s *_standardFS) ReadFile(filename string) ([]byte, error) {
 	return ioutil.ReadFile(filename)
 }
+
+func isNetworkError(err error) bool {
+	_, ok := err.(*net.OpError)
+	return ok
+}
+
+func isRetryablePingError(err error) bool {
+	godockerError, ok := err.(*godocker.Error)
+	if ok && godockerError.Status != http.StatusOK {
```

Comment:
It's more of a future-proofing thing where I don't want us to be impacted by changes in this code base.